src/wscearth/views.py

```diff
@@ -53,11 +53,14 @@ def index():
     return flask.render_template("positions_map.html")
 
 
-@app.route("/api/path/<shortname>")
-def api_path(shortname):
+@app.route("/api/path/<teamnum>")
+@cache.cached(timeout=30)
+@flask_cachecontrol.cache_for(seconds=30)
+def api_path(teamnum):
     """Render JSON path positions for car"""
 
-    query = f'SELECT * FROM "telemetry" WHERE shortname = \'{shortname}\' and time >= -30d'
+    teamnum = int(teamnum)
+    query = f'SELECT * FROM "telemetry" WHERE teamnum = {teamnum} and time >= -30d'
```

Comment:
I'm not sure how much data this will end up pulling down on a regular basis. We may need to think about limiting to the most recent N hours' data. I'll think about how to quantify any performance impact.